<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lh_gui</class>
 <widget class="QDialog" name="lh_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Latency Histogram</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>251</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QPushButton" name="run_lh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Latency Histogram</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Base Thread Options</string>
   </property>
   <widget class="QCheckBox" name="showBase">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Base Thread</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bin Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="baseBinSize">
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Bins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="baseBins">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>184</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Thread Interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="baseThreadInterval">
       <property name="suffix">
        <string>ns</string>
       </property>
       <property name="minimum">
        <number>5000</number>
       </property>
       <property name="maximum">
        <number>50000</number>
       </property>
       <property name="singleStep">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Servo Thread Options</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Bin Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="servoBinSize">
       <property name="minimum">
        <number>25</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>114</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="servoBins">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>199</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Thread Interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="servoThreadInterval">
       <property name="suffix">
        <string>ns</string>
       </property>
       <property name="minimum">
        <number>25000</number>
       </property>
       <property name="maximum">
        <number>2000000</number>
       </property>
       <property name="singleStep">
        <number>5000</number>
       </property>
       <property name="value">
        <number>1000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>LogScale</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>76</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QRadioButton" name="logscaleY">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="logscaleX">
       <property name="text">
        <string>X</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Display Text</string>
   </property>
   <widget class="QLineEdit" name="showText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>341</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="showNotDisplayedBins">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Count of Bins not Displayed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lh_gui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lh_gui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
